src/less/components/card.less

```diff
@@ -68,17 +68,9 @@
                 margin-bottom: 0;
             }
 
-            .material-icons {
-                color: @gray-3;
-            }
-
             .media {
                 .media-body {
-                    p,
-                    small,
-                    h2,
-                    h3,
-                    h4 {
+                    > * {
```

Comment:
Should probably not use the `>` selector here, if you for instance add a `<div>` with a `<p>` inside it won't get the color change. Although I'm unsure of wether or not to this rule should be set by card and not media.

Reply:
Then what should i use, and which usecases should i account for? Right now that selector is just for the way MediaObject is today. I feel a lot of the things surrounding card is a bit vague.